### ISSUE TITLE
SQLException: Access denied for user

### ISSUE THREAD

**Issue by zhangxinjian123** [zendesk/maxwell#1364]
16:21:20,737 ERROR Maxwell - SQLException: Access denied for user 'test'@'192.168.2.3' (using password: YES)

The test user only has read-only permission. Can't Maxwell read binlog?

**Comment by osheroff** [comment]
nope, just like the error message says.  The maxwell user needs the `REPLICATION SLAVE` privilege.
